go-chaos/go.sum

```diff
@@ -128,7 +128,7 @@ github.com/buger/jsonparser v0.0.0-20180808090653-f4dd9f5a6b44/go.mod h1:bbYlZJ7
 github.com/bugsnag/bugsnag-go v0.0.0-20141110184014-b1d153021fcd/go.mod h1:2oa8nejYd4cQ/b0hMIopN0lCRxU0bueqREvZLWFrtK8=
 github.com/bugsnag/osext v0.0.0-20130617224835-0dd3f918b21b/go.mod h1:obH5gd0BsqsP2LwDJ9aOkm/6J86V6lyAXCoQWGw3K50=
 github.com/bugsnag/panicwrap v0.0.0-20151223152923-e2c28503fcd0/go.mod h1:D/8v3kj0zr8ZAKg1AQ6crr+5VwKN5eIywRkfhyM/+dE=
-github.com/camunda-cloud/zeebe/clients/go v1.3.14 h1:7YR0fAfn2b7X0+QR5SUd4heGb+TPY5kSXy9E3529b6M=
+github.com/camunda-cloud/zeebe/clients/go v1.3.14 h1:NjvinpaQIUlQQbfseqWFrCGkFB8SnbrG3fNongTzac8=
```

Comment:
:question: I'm a bit confused about this one, without fixing the checksum I can't download the dependencies. Not sure why the checksum would have changed.

Reply:
Yeah, CI says that the issue is on my side. I've fixed it by cleaning up my $GOPATH. I'll remove the commit from here.

Reply:
Actually nevermind, I still have the same issue after cleaning up $GOPATH. I'll upgrade to 8.1 then.